src/v/cloud_storage_clients/abs_client.cc

```diff
@@ -295,11 +295,11 @@ abs_request_creator::make_list_blobs_request(
   const bucket_name& name,
   bool files_only,
   std::optional<object_key> prefix,
-  [[maybe_unused]] std::optional<object_key> start_after,
-  std::optional<size_t> max_keys,
+  std::optional<size_t> max_results,
```

Comment:
Wondering about validating this with azurite. Would parameterizing some version the cloud self test test on ABS in ducktape (see usages of `CloudStorageType.ABS`) cover any of the new changes?

Reply:
Would a test similar to https://gist.github.com/andrwng/941bf4eca34a020aac59c390ddb9a4c3 exercise the continuation path?

Reply:
Yes, I'm trying to remember what I meant from my previous comment actually. Parameterized the test with `@matrix(cloud_storage_type=get_cloud_storage_type(), remote_read, remote_write)` and all pass :+1:

Reply:
I guess my comment was thinking about ensuring the MinIO bucket is full up before running the self-test, which you have in your gist. Maybe we could further parameterize the test around the number of objects to `PUT` into the MinIO bucket before starting.